appengine-java8/springboot-helloworld/pom.xml

```diff
@@ -20,7 +20,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <spring.boot.version>2.4.3</spring.boot.version> <!-- DO NOT UPDATE w/o MANUAL TESTING -->
+    <spring.boot.version>2.4.4</spring.boot.version> <!-- DO NOT UPDATE w/o MANUAL TESTING -->
```

Comment:
@averikitsch do you know what this manual testing is?

Reply:
Checkout the code and try it manually.  There was a period where some samples, like this didn't adequately test.  I suspect a minor bump is ok.

Reply:
More context, Kokoro is not set up to deploy the sample app. Spring boot on Java 8 has a few work arounds; therefore we should test that it still deploys.